content/docs/react-without-jsx.md

```diff
@@ -23,7 +23,7 @@ ReactDOM.render(
 );
 ```
 
-can be compiled to this code that does not use JSX:
+उपरोक्त कोड को बिना JSX के नीचे दिए गए तरीक़े से भी लिखा जा सकता है।
```

Comment:
`उपरोक्त कोड को बिना JSX के नीचे दिए गए तरीक़े से भी लिखा जा सकता है।` 
=> 
`niche diye code mein compile kiya ja skta hai jismein JSX ka estemal nhi kiya gya hai`

Reply:
I'm using `उपरोक्त कोड को` in front of suggested string, because it seems like we're missing some reference without `उपरोक्त कोड को` in Hindi language.
So, the final string looks like `उपरोक्त कोड को नीचे दिए कोड मे कंपाइल किया जा सकता है जिसमें JSX का उपयोग नहीं किया गया है`